include/ddsrouter/types/address/Address.hpp

```diff
@@ -39,6 +39,7 @@ using PortType = uint16_t;
 //! Different versions allowed for IP
 enum IpVersion
 {
+    NONE_IPv,
```

Comment:
I do not think this new category is really necessary.

Reply:
:+1: